Tests/TokamakCLITests/TokamakLint/TokamakLoggerTest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LoggerTest.swift
+//  TokamakCLITests
+//
+//  Created by Matvii Hodovaniuk on 5/31/19.
+//
+
+import Logging
+@testable import TokamakLint
+import XCTest
+
+final class TokamakLoggerTests: XCTestCase {
+  func testLogToFile() throws {
```

Comment:
there are no assertions in this test, can you please clarify under what conditions this test would fail?